### DISCUSSION TITLE
Bypass build process and pass a bundle to the storybook

### DISCUSSION THREAD

**Discussion by SengitU** [storybookjs/storybook#23094] [Ideas]
**Is your feature request related to a problem? Please describe**
We are building storybook in our CI pipeline in order to have a place that we can check how our standalone components look like. However, we already build a bundle for our application with the webpack before the storybook. I couldn't find an option where I can pass the bundle to the storybook.
To simply describe, I'd like to bypass webpack build step of the storybook and I'd like to pass the current bundle in order to increase our pipeline's efficiency.

**Describe the solution you'd like**
I'd like to pass pre-built bundle to the storybook and I'd like to bypass storybook's webpack build.

**Describe alternatives you've considered**
I've checked the current documentation and couldn't find a proper way to achieve this. I've also tried to filter issues by the keyword `bundle` and `webpack` but I couldn't find something helpful.

**Are you able to assist to bring the feature to reality?**
Yes, I'd love to

**Additional context**
Please let me know if this is possible, or please let me know if that's not possible and I'm mistaken horribly 😄

**Comment by reno1979** [comment]
This is almost the same as our requirements. We have to analyse the vanilla webcomponents before the webpack build process starts. 

Using the plugin webpack hooks I am able to prebuild the required files (the webcomponent analyzer and the test bundel). 

Every time the HMR process starts, the plugin starts a custom script tot build and move the required files. More details about the plugin: https://github.com/tienne/webpack-hook-plugin

**Comment by shilman** [comment]
Converting this to a discussion. We’re reserving feature issues for things that are on our [short-term roadmap](https://github.com/storybookjs/storybook/milestones). We’ll check back periodically to see whether this is a good candidate to be added. If this feature would be useful to you, please upvote!
